fighthealthinsurance/common_view_logic.py

```diff
@@ -767,20 +768,25 @@ def find_next_steps(
                 existing_answers["date of service"] = date_of_service
             if "date_of_service" not in existing_answers:
                 existing_answers["date_of_service"] = date_of_service
+        # This is unique to professional so using this for now to help specialize questions
+        prof_pov = denial.professional_to_finish
         if in_network is not None:
-            denial.provider_in_network = in_network
+            denial.provider_inz_network = in_network
+            # If they know about in_network they are definitely a professional
+            prof_pov = True
```

Comment:
So this seems like not the right way to determine if an appeal is professional or not.

Reply:
I found that if that in_network exists then that is a double check on making it clear this is prof_pov - left it in to catch anything that falls past the creating_professional (there may be some weird case we missed for how that gets/doesn't get set).